src/components/collaboration/CollaborativeNotes.tsx

```
@@ -0,0 +1,253 @@
+    const newTitle = e.target.value;
+    setTitle(newTitle);
+    onDocumentChange({
+      ...document,
+      title: newTitle,
+      version: document.version + 1,
+      updatedAt: new Date().toISOString(),
+      lastEditor: currentUser.name,
+    });
+  };
+
+  const insertFormatting = (prefix: string, suffix: string = '') => {
```

ESLint

[error] Replace `''` with `""`

(prettier/prettier)